docs/tool_jcmd.md

```diff
@@ -29,15 +29,22 @@ Use the `jcmd` tool to run diagnostic commands on a specified VM.
 
 The command syntax is as follows:
 
-    jcmd [<options>] [<vmid> <arguments>]
+    jcmd [<options>] [<vmid | display name | 0> <arguments>]
 
 Where:
 
-- The available `<options>` are:    
+- The available `<options>` are:
```

Comment:
Pls remove `on` in `therefore on specifying`

Reply:
Done. Thanks!